### DISCUSSION TITLE
Allow the editor Test feature to start where your current timeline time is.

### DISCUSSION THREAD

**Discussion by Coppertine** [ppy/osu#15619] [Ideas] [answered]
Basically, if you press the "Test" button, the beatmap testing starts from the beginning and not at the current timeline position.
If possible, either doing a SHIFT + Click or just a regular click would make the testing start from your current time/position.

**Comment by frenzibyte** [comment, marked as answer]
Will be resolved in https://github.com/ppy/osu/pull/15609.
